test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -1981,7 +1992,7 @@ describe('isPendingState()', () => {
   });
 
   it('should return "false" for non-pending states', () => {
-    expect(isPendingState(undefined)).toBe(false);
+    expect(isPendingState(undefined as any)).toBe(false);
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9637746)
```suggestion
    expect(isPendingState(undefined as unknown)).toBe(false);
```